db/migrate/20150828150524_remove_approval_groups.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveApprovalGroups < ActiveRecord::Migration
+  def change
+    drop_table :approval_roles
+    drop_table :approval_groups
+    drop_table :approval_groups_users
+  end
+end
```

Comment:
the evolution of this branch required it be here. the original table was `user_roles` which we renamed to `approval_roles` and then realized it wasn't used at all. The user_roles namespace was what we wanted to use.

We can pull out this particular migration into another PR, but I considered it a 'house-cleaning' prior to the new roles' tables structures.